module/VuFind/src/VuFind/AjaxHandler/GetSearchResults.php

```diff
@@ -0,0 +1,424 @@
+<?php
+
+/**
+ * "Get Search Results" AJAX handler
+ *
+ * PHP version 7
```

Comment:
We should change all `PHP version 7` comments to `PHP version 8` at this point.

Reply:
Oops, fixed.